### DISCUSSION TITLE
[BUG] Corner icon selection finicky

### DISCUSSION THREAD

**Discussion by landoncope** [rxhanson/Charmstone-Community#37] [Q&A]
![2023-09-06_08-54-26 (1)](https://github.com/rxhanson/Charmstone-Community/assets/5022641/f5d411b5-a1fb-4c57-be25-a08b7505ec69)

The corner icon selection is sometimes finicky. The app seems to prefer the side icons at a certain distance. I've noticed it throws my muscle memory off a bit when the wrong application opens. 

Using Charmstone 0.30. I've tried different Safe Area and Min Dist to Appear values with no luck.

**Comment by rxhanson** [comment]
Thanks for reporting. What's going on here is that the UI doesn't line up with the actual angle of where the cursor is relative to the start point. Charmstone sort of corrects itself by selecting the item when the cursor enters its grid space. In my usage, I guess I just move the cursor a little farther so as to avoid this small ambiguous area. Next time I'm in this part of the code, I'll see if I can get rid of that small area of ambiguity.
